components/suggest/src/store.rs

```diff
@@ -2184,15 +2195,21 @@ mod tests {
         store.ingest(SuggestIngestionConstraints::all_providers());
         assert_eq!(
             store.fetch_suggestions(SuggestionQuery::fakespot("simp")),
-            vec![simpsons_suggestion()],
+            vec![simpsons_suggestion(
+                "The <b>Simpsons</b>: Skinner's Sense of Snow (DVD)"
+            )],
```

Comment:
One interesting aspect of the feature is that it highlights the entire term, not just the part that was matched.  So here it highlights "Simpsons", even though the user input was "simp".